projects/Mallard/src/components/icons/__tests__/Newspaper.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import TestRenderer, { ReactTestRendererJSON } from 'react-test-renderer'
+import { Newspaper } from '../Newspaper'
+
+describe('Newspaper', () => {
+    it('should display a Newspaper icon in SVG', () => {
```

Comment:
this feels like a pretty simple component. what's the emotivation for adding a unit test?

Reply:
Discussed in person: this could help catch someone accidentally changing the colour of this icon whilst trying to change something else